### ISSUE TITLE
Bug: `await-fire-event` doesn't show the fireEvent method correctly in the error message

### ISSUE THREAD

**Issue by Belco90** [testing-library/eslint-plugin-testing-library#590]
### Have you read the Troubleshooting section?

Yes

### Plugin version

v5.5.0

### ESLint version

v8.14.0

### Node.js version

v16.15.0

### package manager and version

npm v8.9.0

### Operating system

macOS v12.3.1

### Bug description

Errors reported by `await-fire-event` rule are not indicating the `fireEvent` method name correctly in the message.

### Steps to reproduce

```js
test('increments value on click', async () => {
  render(App);

  expect(screen.getByText('Times clicked: 0')).toBeDefined();

  const button = screen.getByText('increment');

  fireEvent.click(button);

  expect(screen.getByText('Times clicked: 1')).toBeDefined();
});
```

### Error output/screenshots

![CleanShot 2022-05-20 at 00 14 50](https://user-images.githubusercontent.com/2677072/169413320-39596c6b-27af-44f7-8d4f-8f248ef35cea.png)

### ESLint configuration

N/A

### Rule(s) affected

`await-fire-event`

### Do you want to submit a pull request to fix this bug?

No

**Comment by IvanAprea** [comment]
Hello, can i propose a solution for this?

**Comment by Belco90** [comment]
@IvanAprea yes please! PRs are more than welcome.
